src/styles/globals.css

```diff
@@ -17,7 +17,7 @@ body {
     padding: 0;
     margin: 0;
     height: 100%;
-    font-family: var(--font-quicksand);
+    font-family: var(--font-quicksand) !important;
```

Comment:
also this change, have no idea why the styles are included in a different order in debug vs release mode